pkgs/applications/networking/remote/x2goclient/default.nix

```diff
@@ -29,7 +29,9 @@ stdenv.mkDerivation rec {
 
   installTargets = [ "install_client" "install_man" ];
   postInstall = ''
-    wrapProgram "$out/bin/x2goclient" --suffix PATH : "${nx-libs}/bin:${openssh}/libexec";
+    wrapProgram "$out/bin/x2goclient" \
+      --suffix PATH : "${nx-libs}/bin:${openssh}/libexec" \
+      --prefix QT_PLUGIN_PATH : ${qtbase}/${qtbase.qtPluginPrefix}
```

Comment:
I think wrapping Qt applications like that was deprecated with [this PR](https://github.com/NixOS/nixpkgs/pull/54525), so the way to do it would be to have `wrapQtAppsHook` in `nativeBuildInputs`.

Reply:
Or use qt5.mkDerivation instead of stdenv.mkDerivation, to get automatic wrapping (no need to add wrapQtAppsHook), if I understand correctly.

Reply:
Yes, much cleaner. Fixed.